db/seeds/warning_templates/off-topic.md

```diff
@@ -0,0 +1,9 @@
+We've noticed that you've recently been writing a number of posts about topics that are beyond the scope of this site.

```

Comment:
"recently been writing" -> "written".  Even if it was months ago and we only just noticed, we want to be able to send this message.

Reply:
What @cellio said :)